src/conversions/either.rs

```diff
@@ -67,12 +67,40 @@ where
 }
 
 #[cfg_attr(docsrs, doc(cfg(feature = "either")))]
-impl<'py, L, R, E1, E2> IntoPyObject<'py> for Either<L, R>
```

Comment:
👍 nice tidy up

Reply:
Yeah, not sure why made this so overly complicated the first time 😅